util/plugins/eu.esdihumboldt.util/src/eu/esdihumboldt/util/geometry/interpolation/Interpolation.java

```diff
@@ -33,30 +33,31 @@
 	/**
 	 * Grid step size factor
 	 */
-	protected final int GRID_FACTOR = 2;
+	protected static final int GRID_FACTOR = 2;
 	/**
 	 * Next coordinate distance factor.
 	 */
-	protected final double NEXT_COORDINATE_DISTANCE_FACTOR = 4;
+	protected static final double NEXT_COORDINATE_DISTANCE_FACTOR = 4;
+
 	/**
-	 * Distance of next coordinate from current coordinate
+	 * Rounding scale for grid cell double value
 	 */
-	protected final double NEXT_COORDINATE_DISTANCE;
+	private static final int ROUNDING_SCALE = 6;
```

Comment:
Would be much preferable to use the respective constant here, or even the enum value instead of this magic number.